ui/app/components/clients/page/overview.ts

```diff
@@ -4,5 +4,13 @@
  */
 
 import ActivityComponent from '../activity';
+import { service } from '@ember/service';
+import type FlagsService from 'vault/services/flags';
 
-export default class ClientsOverviewPageComponent extends ActivityComponent {}
+export default class ClientsOverviewPageComponent extends ActivityComponent {
+  @service declare readonly flags: FlagsService;
+
+  get hasSyncHistoryOrIsActivated() {
+    return this.args.activity.total.secret_syncs > 0 || this.flags.secretsSyncIsActivated;
```

Comment:
🤔 Hmm, maybe I wasn't clear in my previous comments. But 0 clients in this context is still an important metric so we should remove the first part of this conditional.

Reply:
oh oh oh. yes, i understand what you mean. should we check if `secret_syncs !== undefined` then? or `!== null`? let's chat in slack about this.